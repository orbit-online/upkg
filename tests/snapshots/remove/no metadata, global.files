.
└── lib
    └── upkg
        └── upkg.json

2 directories, 1 file
