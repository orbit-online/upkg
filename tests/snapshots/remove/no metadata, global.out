upkg: Removed 'acme-no-metadata'
